vhdbuilder/packer/configure-windows-vhd.ps1

```diff
@@ -12,7 +12,7 @@ $ErrorActionPreference = "Stop"
 
 filter Timestamp {"$(Get-Date -Format o): $_"}
 
-$global:containerdPackageUrl = "https://marosset.blob.core.windows.net/pub/containerd/containerd-0.0.87-public.zip"
+$global:containerdPackageUrl = "https://k8swin.blob.core.windows.net/k8s-windows/containerd/containerplat-aks-test-0.0.8.zip"
```

Comment:
Also cache https://aksenginee2etestimages.blob.core.windows.net/test-content/windows-cni-containerd.zip

Reply:
windows-cni-containerd.zip is only needed for kubenet (although it would not cause any harm to cache it too)

Reply:
Thanks for the message and close this as we don't support Windows Kubenet.